test/include/aes_keywrap_test.h

```diff
@@ -0,0 +1,170 @@
+/**
+ * @file  aes_keywrap_test.h
+ *
+ * Provides unit tests for the pelz AES keywrap (RFC-3394 - no padding)
+ */
+
+#ifndef AES_KEYWRAP_TEST_H
+#define AES_KEYWRAP_TEST_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+/**
+ * Specify maximum number of test vector sets (vector files) that can be
+ * contained within a "vector set compilation" (used to size that array).
+ */
+#define MAX_VECTOR_SETS_IN_COMPILATION 20
+
+/**
+ * Specify maximum number of test vectors to process when parsing
+ * a test vector file.
+ */
+#define MAX_KEYWRAP_TEST_VECTOR_COUNT 500
+
+/**
+ * Specify the maximum length (in chars) of a test vector component
+ * This is needed to appropriately size the buffers used to parse and
+ * process test vector components read from a file. For example, a
+ * value of 2176 (2048 + 128) supports up to a 2048 hexadecimal character
+ * string that can specify a 1024-byte or 8192-bit test vector component,
+ * as well as up to 128 leading/trailing characters.
+ */
+#define MAX_TEST_VECTOR_COMPONENT_LENGTH 2176
```

Comment:
Same question here, if all these are based on what we know about the NIST vector sets we should add that as a comment.